cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/rest/CloudControllerRestClientImpl.java

```diff
@@ -1408,6 +1411,22 @@ public Upload getUploadStatus(UUID packageGuid) {
                               .build();
     }
 
+    @Override
+    public List<CloudPackage> getPackagesForApplication(UUID applicationGuid) {
+        return getPackagesForApplication(applicationGuid, null);
```

Comment:
Instead of `null` pass an instance of `NoRequestOrdererV3` here. That would make the `determineRequestOrderer` method redundant.

Reply:
+1